internal/domain/human.go

```diff
@@ -60,16 +61,30 @@ func (f Gender) Specified() bool {
 	return f > GenderUnspecified && f < genderCount
 }
 
-func (u *Human) IsValid() bool {
-	return u.Username != "" && u.Profile != nil && u.Profile.IsValid() && u.Email != nil && u.Email.IsValid() && u.Phone == nil || (u.Phone != nil && u.Phone.PhoneNumber != "" && u.Phone.IsValid())
+func (u *Human) Normalize() error {
```

Comment:
No differentiation between normalization and validation?

Reply:
To make it explicit that state is changed, I renamed the method. But I didn't want to change existing logic where the effects are not obvious.
Instead I created a follow-up issue that addresses this https://github.com/zitadel/zitadel/pull/5382#discussion_r1131261119

Reply:
What I meant is that the Normalize functions for the different structs handle differently, which shouldn't be the case right?

Reply:
It shouldn't, thats why there is a follow up issue. It is already wrong on main.